packages/presets/src/tests/test_erc721.cairo

```diff
@@ -1126,8 +1126,8 @@ fn test_v2_missing_camel_selector() {
     let selector = selector!("ownerOf");
     let expected_panic_message = format!(
         "Entry point selector {} not found in contract {}",
-        selector.into_base_16_string(),
-        v1.contract_address.into_base_16_string()
+        selector.into_base_16_string_no_padding(),
+        v1.contract_address.into_base_16_string_no_padding()
     );
```

Comment:
We can use `utils::assert_entrypoint_not_found_error` here by removing `.unwrap_err()` on the 1124th line and providing the call result to the function

Reply:
Yeah, this makes it more readable and consistent with the other tests. Added!